packages/apps/spaces/ui/form/Select/Select.tsx

```diff
@@ -146,7 +146,7 @@ const getDefaultClassNames = ({
         : 'mt-2 animate-slideDownAndFade',
     ),
   menuList: () =>
-    'p-2 max-h-[300px] border border-gray-200 bg-white outline-offset-[2px] outline-[2px] rounded-lg shadow-lg overflow-y-auto overscroll-auto',
+    'p-2 max-h-[300px] border border-gray-200 bg-white outline-offset-[2px] outline-[2px] rounded-lg shadow-lg overflow-y-auto overscroll-auto min-w-[120px]',
```

Comment:
Why this change? I've implemented after figma specs this component last week or so. I did not fond any issue regarding width not functioning ok.

Reply:
it was too narrow for every select in new form (new selects are very narrow and select without min width were using width of select input - therefore causing problems
)